revscoring/languages/spanish.py

```diff
@@ -183,6 +183,7 @@
     r"sida",  # AIDS
     r"sorete",  # piece of shit
     r"sudaca",  # greaser
+    r"subnormal", # retarded
```

Comment:
Needs another space here too.  Then all should be good :)

Reply:
Fixing those. Thanks.